src/components/Counter.js

```diff
@@ -1,59 +1,54 @@
 import React, { Component } from "react";
-import { connect } from 'react-redux';
-import { increment, decrement } from '../actions';
+import { connect } from "react-redux";
+import { increment, decrement, oddincrement } from "../actions";
 
 class Counter extends Component {
-    incrementIfOdd = () => {
-        // Stretch Problem: Implement an increment function that
-        // only increments if the counter value is odd
-    };
+  incrementIfOdd = () => {
+    // Stretch Problem: Implement an increment function that
+    // only increments if the counter value is odd
+    console.log('odd increment clicked')
+    if(this.props.count % 2 !== 0) this.props.oddincrement();
```

Comment:
You have your check if the count is odd, so if it passes that check, you can just call `this.props.increment()`, no need for another method here.